packages/ckeditor5-ckbox/src/ckboxcommand.ts

```diff
@@ -127,24 +127,57 @@ export default class CKBoxCommand extends Command {
 	 * - tokenUrl The token endpoint URL.
 	 * - serviceOrigin The base URL of the API service.
 	 * - forceDemoLabel Whether to force "Powered by CKBox" link.
-	 * - dialog.onClose The callback function invoked after closing the CKBox dialog.
 	 * - assets.onChoose The callback function invoked after choosing the assets.
+	 * - dialog.onClose The callback function invoked after closing the CKBox dialog.
+	 * - dialog.width The dialog width in pixels.
+	 * - dialog.height The dialog height in pixels.
+	 * - categories.icons Allows setting custom icons for categories.
+	 * - view.openLastView Sets if the last view visited by the user will be reopened
+	 * on the next startup.
+	 * - view.startupFolderId Sets the ID of the folder that will be opened on startup.
+	 * - view.startupCategoryId Sets the ID of the category that will be opened on startup.
+	 * - view.hideMaximizeButton Sets whether to hide the ‘Maximize’ button.
+	 * - view.componentsHideTimeout Sets timeout after which upload components are hidden
+	 * after completed upload.
+	 * - view.dialogMinimizeTimeout Sets timeout after which upload dialog is minimized
+	 * after completed upload.
 	 */
 	private _prepareOptions() {
 		const editor = this.editor;
 		const ckboxConfig = editor.config.get( 'ckbox' )!;
 
+		const dialog = ckboxConfig.dialog!;
+		const categories = ckboxConfig.categories!;
+		const view = ckboxConfig.view!;
+		const upload = ckboxConfig.upload!;
+
```

Comment:
Why not-null assertions (`!`)? Below, there are guards (like `dialog && dialog.with`) for each of those values.

Reply:
I don't know certainly why, but I afraid `ckboxConfig.dialog!.width!` is not valid. I tried it out and tests are crushed with that expression.